docs/core/diagnostics/sos-debugging-extension.md

```diff
@@ -157,10 +156,10 @@ However the common commands have been aliased so that you don't need the `sos` p
 |    `loadsymbols`                      | Load the .NET Core native module symbols.
 |    `name2ee`                          | Displays the `MethodTable` and `EEClass` structures for the specified type or method in the specified module.
 |    `pe`                               | Displays and formats fields of any object derived from the <xref:System.Exception> class at the specified address.
-|    `setclrpath`                       | Set the path to load coreclr dac/dbi files: `setclrpath <path>`
+|    `setclrpath`                       | Sets the path to load coreclr dac/dbi files. `setclrpath <path>`
 |    `sethostruntime`                   | Sets or displays the .NET Core runtime directory to use to run managed code in SOS.
 |    `setsymbolserver`                  | Enables the symbol server support.
-|    `setsostid`                        | Set the current OS tid/thread index instead of using the one lldb provides: `setsostid <tid> <index>`
+|    `setsostid`                        | Set the current OS tid/thread index instead of using the one lldb provides. `setsostid <tid> <index>`
```

Comment:
```suggestion
|    `setsostid`                        | Sets the current OS tid/thread index instead of using the one lldb provides. `setsostid <tid> <index>`
```